commands/aliases.go

```diff
@@ -96,44 +110,41 @@ func aliasCommand(ctx context.Context, args []string) (string, error) {
 		p := dgwidgets.NewPaginator(
 			ctx.Value(messageContextValue("Session")).(*discordgo.Session),
 			ctx.Value(messageContextValue("ChannelID")).(string))
+
+		var sortedAliases []string
+		for a := range aliasMap {
+			sortedAliases = append(sortedAliases, a)
+		}
+		sort.Strings(sortedAliases)
+
+		var aliasListing []string
+		for i, a := range sortedAliases {
+			aliasListing = append(aliasListing, fmt.Sprintf("%d) **%s**", i+2, a))
+		}
+
 		p.Add(&discordgo.MessageEmbed{
 			Color: 0,
 
 			Fields: []*discordgo.MessageEmbedField{
 				{
-					Name: "Aliases",
-					Value: func() string {
-						var sortedAliases []string
-						for a := range aliasMap {
-							sortedAliases = append(sortedAliases, fmt.Sprintf("**%s**", a))
-						}
-						sort.Strings(sortedAliases)
-						for i := range sortedAliases {
-							sortedAliases[i] = fmt.Sprintf("%d) %s", i+2, sortedAliases[i])
-						}
-						return strings.Join(sortedAliases, "\n")
-					}(),
+					Name:  "Aliases",
+					Value: strings.Join(aliasListing, "\n"),
 				},
 			},
 		})
 
-		for aliasName, aliasValue := range aliasMap {
+		for _, aliasName := range sortedAliases {
+			info := aliasMap[aliasName]
 			embed := &discordgo.MessageEmbed{
 				Title:       aliasName,
-				Description: aliasValue,
+				Description: info.Value,
```

Comment:
As a thought, do we wanna hide the potentially ugly URL if its an image?

Reply:
I wanted to do that but then if the links are acting weird like that gify one from earlier, it'll show nothing at all :/

Reply:
Fair enough, we can always review it at a later stage to iron it out